services/frontend/src/components/Sidebar/SidebarLayout.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Scrollspy from "react-scrollspy";
+import "./Sidebar.scss";
+
+const SidebarLayout = ({sections, children, containerClass}) => {
```

Comment:
Changed the sections to "scrollSpySelectors" when passed to the sidebar, I think leaving `sections` as is makes sense though because I'm actually using the semantic HTML element `<section>` .

I could also go with like sectionData maybe?